Jenkinsfile

```diff
@@ -182,9 +182,9 @@ pipeline {
             description: 'Docker organization to publish to',
         )
         booleanParam(
-            name: 'USE_SLES_STEMCELL',
+            name: 'USE_SLES_BASE',
```

Comment:
We should be consistent here and call it `USE_SLE_BASE`. SLE is the SUSE Linux Enterprise code stream from which various products are built. SLES is the server product. We are close to the server, but it's not really the SLES product we use so SLE is more accurate. We also use `sle` in other places such as the user-visible name of the stack.